algorithms/automorphism_problem.py

```diff
@@ -0,0 +1,26 @@
+from supporting_components.basicpermutationgroup import FindNonTrivialOrbit, Stabilizer, Orbit
+from supporting_components.permv2 import permutation
+from typing import List
+
+
+def order_computation(H: 'List[permutation]'):
+    """
+    Recursively computes the order of a list of permutations.
+    Based on slide 21 of lecture 4 and notes.
+    :param H: Permutation group
+    :return: int with the order of the list of permutations
+    """
+    # Choose a nontrivial orbit
+    a = FindNonTrivialOrbit(H)
+    # Determine orbit of nontrivial orbit
+    O_H = Orbit(H, a)
+
+    # End of recursion, |H| = |O_H| * 1
```

Comment:
Deze comment is denk ik een beetje misleidend. Je eindigt de recursie omdat |H| 1 is, en dat betekent niet per se dat |O_H| * 1 ook 1 is.

Reply:
Ik heb de documentatie wat aangepast, iets uitgebreider.

En heb op youtube een uitleg kunnen vinden waarom de order eigenlijk inderdaad een vermenigvuldiging van orbit lengtes is. https://youtu.be/AnJOjE8nVFY?t=1001

En in theorie is het zo dat zodra de lengte van de stabilizer H_0 1 is, je kan stoppen. Maar de order is dan de lengte van de orbit van de nieuwe stabilizer en daarom moet je nog 1 recursie stap verder en daar kun je dan zeggen dat de order alleen de lengte van de orbit is.